src/task/builder.rs

```diff
@@ -8,6 +9,11 @@ use pin_project_lite::pin_project;
 use crate::io;
 use crate::task::{JoinHandle, Task, TaskLocalsWrapper};
 
+#[cfg(not(target_os = "unknown"))]
+thread_local! {
+    static IS_CURRENT_BLOCKING: Cell<usize> = Cell::new(0);
```

Comment:
Hmm, not sure I love this name. Something like `BLOCK_ON_ENABLED` could work better. Or perhaps it's just missing docs.

Reply:
fixed